.packit.yaml

```diff
@@ -0,0 +1,40 @@
+---
+downstream_package_name: python-logdetective
+upstream_package_name: logdetective
+specfile_path: python-logdetective.spec
+upstream_tag_template: 'v{version}'
+srpm_build_deps:
+- poetry
+actions:
+  post-upstream-clone:
+    # rawhide specfile will be the source of truth: we don't need to have the specfile upstream
+    - curl -Lvso python-logdetective.spec https://src.fedoraproject.org/rpms/python-logdetective/raw/main/f/python-logdetective.spec
+  create-archive:
+    - poetry build -f sdist
+    - bash -c "ls -1t ./dist/*.tar.gz | head -n 1"
+  get-current-version:
+    - poetry version -s
+
+jobs:
+  - job: propose_downstream
```

Comment:
Do we want it? I have to admit that I am more happier with pull-from-upstream.

Reply:
good point, I'll merge both packit.yaml configs together